tests/main/interfaces-opengl-nvidia/task.yaml

```diff
@@ -22,13 +22,29 @@ prepare: |
         echo "canary-triplet" >> /usr/lib/$(dpkg-architecture -qDEB_HOST_MULTIARCH)/libnvidia-glcore.so.123.456
         echo "canary-triplet" >> /usr/lib/$(dpkg-architecture -qDEB_HOST_MULTIARCH)/tls/libnvidia-tls.so.123.456
         echo "canary-triplet" >> /usr/lib/$(dpkg-architecture -qDEB_HOST_MULTIARCH)/libnvidia-tls.so.123.456
+        if [[ "$(uname -m)" == x86_64 ]]; then
+            mkdir -p /usr/lib/$(dpkg-architecture -qDEB_HOST_MULTIARCH -ai386)/tls
+            echo "canary-32-triplet" >> /usr/lib/$(dpkg-architecture -qDEB_HOST_MULTIARCH -ai386)/libGLX.so.0.0.1
```

Comment:
This also works with space-separated arguments which I find more readable.

Reply:
Unfortunately IIRC it does not on 14.04 (at least not `-a i386`).

Reply:
```
google:ubuntu-14.04-64 .../tests/main/snap-userd-desktop-app-autostart# dpkg-architecture -qDEB_HOST_MULTIARCH -a i386
dpkg-architecture: unknown option `i386'

Use --help for program usage information.
google:ubuntu-14.04-64 .../tests/main/snap-userd-desktop-app-autostart# dpkg-architecture -qDEB_HOST_MULTIARCH -ai386
i386-linux-gnu
google:ubuntu-14.04-64 .../tests/main/snap-userd-desktop-app-autostart# dpkg-architecture -q DEB_HOST_MULTIARCH -ai386
dpkg-architecture: error:  is not a supported variable name
google:ubuntu-14.04-64 .../tests/main/snap-userd-desktop-app-autostart# dpkg-architecture -qDEB_HOST_MULTIARCH -ai386
i386-linux-gnu
```